### PR TITLE
Update mashtree to 2.0

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`mashtree`](https://bioconda.github.io/recipes/mashtree/README.html): **1.2.0** &rarr; **2.0**

[install with bioconda](http://bioconda.github.io/recipes/mashtree/README.html) [Conda](https://anaconda.org/bioconda/mashtree/files)

Info | Link
-----|-----
Recipe | [`recipes/mashtree`](https://github.com//bioconda/bioconda-recipes/tree/bump/mashtree/recipes/mashtree) (click to view/edit other files)
Releases |[https://github.com/lskatz/mashtree/tags](https://github.com/lskatz/mashtree/tags)
Author | `@lskatz`
***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Comment by rpetit3** [comment]
@lskatz Looking at the release notes for Mashtree v2 (https://github.com/lskatz/mashtree/releases/tag/v2.0), do you want to hold off on merging this (pending recipe update)?

I'm asking because this: 
*This is in alpha and not tested at all. Use at your own risk. (but feedback is always welcome!)*

If merged v2.0 would become the default install for Mashtree

**Comment by lskatz** [comment]
Thank you for alerting me @rpetit3. Correct - please do not have this as the default version.

**Review by rpetit3 — CHANGES_REQUESTED** [review]
@lskatz when you think its ready, ping me. 

I'm not requesting a change, just adding this to require my approval to prevent accidental merging.

**Pull request closed without merging**
